MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentComponent/PaymentComponentsController.swift

```diff
@@ -191,17 +193,22 @@ public final class PaymentComponentsController: PaymentComponentsProtocol {
             completion(nil, nil)
             return
         }
-        let vc = PaymentReviewViewController.instantiate(with: self.giniMerchant,
-                                                         data: nil,
-                                                         paymentInfo: paymentInfo,
-                                                         selectedPaymentProvider: selectedPaymentProvider,
-                                                         trackingDelegate: trackingDelegate,
-                                                         paymentComponentsController: self)
-        completion(vc, nil)
+
+        paymentReviewViewController = PaymentReviewViewController.instantiate(with: self.giniMerchant,
+                                                                              data: nil,
+                                                                              paymentInfo: paymentInfo,
+                                                                              selectedPaymentProvider: selectedPaymentProvider,
+                                                                              trackingDelegate: trackingDelegate,
+                                                                              paymentComponentsController: self)
+        completion(paymentReviewViewController, nil)
     }
 
     // MARK: - Bottom Sheets
 
+    public func paymentInfoBottomSheet() -> UIViewController {
```

Comment:
Suggestion: I would keep using: `loadPaymentReviewScreenWithoutDocument` and/or if mandatory, I would rename this function to something like: `paymentReviewBottomSheet`.

We already have `paymentInfoViewController`, which is about something else. It might be misleading.